### DISCUSSION TITLE
How to link data from table? [May 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5666] [Discord Threads]
> 🙋‍♂️ **Rowin**  
## How do I link the data from the reviewsInput and reviewsOutput tables? I want to automatically fill in the reviewId and commentQuestion columns in the reviewsOutput table with data from the reviewsInput table, which receives data from an API. I'll only manually enter data in the commentAnswer column. Anyone can help me with this basic thing? 🙂  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1095082768368681055/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **Rowin**  
####   
![image0](https://cdn.discordapp.com/attachments/1095082768830046289/1095083133130522684/image.png)  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### refer here   
#### https://docs.nocodb.com/setup-and-usages/link-to-another-record  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1095082768830046289)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
